Source/DafnyCore/AST/Types/TypeParameter.cs

```diff
@@ -197,6 +198,11 @@ public TypeParameter(RangeToken tok, Name name, int positionalIndex, ParentType
     }
   }
 
+  public override SymbolKind? Kind => SymbolKind.TypeParameter;
+  public override string GetDescription(DafnyOptions options) {
+    return null; // TODO test the effect of this
```

Comment:
I'm not sure how to interpret this comment. Have you tested calling `GetDescription` on a `TypeParameter`?